packages/x-charts-pro/src/ResponsiveChartContainerPro/ResponsiveChartContainerPro.tsx

```diff
@@ -17,12 +17,10 @@ const ResponsiveChartContainerPro = React.forwardRef(function ResponsiveChartCon
   const { width: inWidth, height: inHeight, ...other } = props;
   const [containerRef, width, height] = useChartContainerDimensions(inWidth, inHeight);
 
-  useLicenseVerifier('x-charts-pro', releaseInfo);
```

Comment:
This could effectively be moved in the the `ChartContainerPro`, but I do not see where you added it back

Reply:
Oops. I was tinkering with the license things 😅

Reply:
Though the `Watermark` component runs it already, would we still need it?

```tsx
export function Watermark(props: WatermarkProps) {
  const { packageName, releaseInfo } = props;
  const licenseStatus = useLicenseVerifier(packageName, releaseInfo);
```

Reply:
> the Watermark component runs it already

I did not know

> would we still need it

Could be nice in the container since it does not have Watermark, and adding the watermark could add complication

Reply:
done